stochastic_physics/stochastic_physics_wrapper.F90

```diff
@@ -109,7 +109,7 @@ subroutine stochastic_physics_wrapper (GFS_Control, GFS_Data, Atm_block, ierr)
             GFS_Control%input_nml_file, GFS_Control%fn_nml, GFS_Control%nlunit, xlon, xlat, GFS_Control%do_sppt, GFS_Control%do_shum, &
             GFS_Control%do_skeb, GFS_Control%lndp_type, GFS_Control%n_var_lndp, GFS_Control%use_zmtnblck, GFS_Control%skeb_npass,     &
             GFS_Control%lndp_var_list, GFS_Control%lndp_prt_list,    &
-            GFS_Control%n_var_spp, GFS_Control%spp_var_list, GFS_Control%spp_prt_list, GFS_Control%do_spp,                            &
+            GFS_Control%n_var_spp, GFS_Control%spp_var_list, GFS_Control%spp_prt_list, GFS_Control%spp_stddev_cutoff, GFS_Control%do_spp,                            &
```

Comment:
This has a different order than what is in stochastic_physics/stochastic_physcs.F90
here: 
GFS_Control%spp_var_list, GFS_Control%spp_prt_list, GFS_Control%spp_stddev_cutoff,

in stochastic_physics.F90:
spp_var_list_out, spp_stddev_cutoff_out, spp_prt_list_out

Reply:
@JeffBeck-NOAA would you please reply to Phil?

Reply:
Looks like in stochastic_physics.F90, it has been changed to 
spp_var_list_out,  spp_prt_list_out, spp_stddev_cutoff_out